b2sdk/v1/api.py

```diff
@@ -49,3 +53,75 @@ def check_bucket_restrictions(self, bucket_name):
     def cancel_large_file(self, file_id: str) -> FileVersionInfo:
         file_id_and_name = super().cancel_large_file(file_id)
         return file_version_info_from_id_and_name(file_id_and_name, self)
+
+    @overload
+    def download_file_by_id(
+        self,
+        file_id: str,
+        download_dest: AbstractDownloadDestination,
+        progress_listener: Optional[v2.AbstractProgressListener] = None,
+        range_: Optional[Tuple[int, int]] = None,
+        encryption: Optional[v2.EncryptionSetting] = None,
+        allow_seeking: bool = True,
+    ) -> dict:
+        ...
+
+    @overload
```

Comment:
ok - after I understood how it works I like this :) 

it is also kinda nice that we support in `v1` both old `download_dest` interface and new `DownloadedFile`

Reply:
also - here I see that `allow_seeking` here allows support of unseekable destinations in `v1`

Reply:
yes, but it's BC and generally beneficial, so it's ok, right?